lib/Controller/WopiController.php

```diff
@@ -195,10 +195,13 @@ public function checkFileInfo($fileId, $access_token) {
 		}
 
 		if ($this->shouldWatermark($isPublic, $wopi->getEditorUid(), $fileId, $wopi)) {
+			$email = $user !== null && !$isPublic ? $user->getEMailAddress() : "";
```

Comment:
Looks good, but $isPublic is a confusing name (I know it's not added in this PR).